.github/workflows/tft.yml

```
@@ -0,0 +1,194 @@
+          - platform: CentOS-Stream-8
+            ansible_version: "2.9"
+          # On CentOS-Stream-8, latest supported Ansible is 2.16
+          - platform: CentOS-Stream-8
+            ansible_version: "2.16"
+          - platform: CentOS-Stream-9
+            ansible_version: "2.17"
+          - platform: CentOS-Stream-10
+            ansible_version: "2.17"
+    runs-on: ubuntu-latest
+    env:
+      ARTIFACTS_DIR_NAME: "tf_${{ github.event.repository.name }}-${{ github.event.issue.number }}_\
```

actionlint (1.7.12)

[error] property "datetime" is not defined in object type {head_sha: string; memory: string; supported_platforms: string}

(expression)